[2022-10-28 21:44:49,344] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: get_files_cetram.extract scheduled__2022-09-01T00:00:00+00:00 [queued]>
[2022-10-28 21:44:49,349] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: get_files_cetram.extract scheduled__2022-09-01T00:00:00+00:00 [queued]>
[2022-10-28 21:44:49,349] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-10-28 21:44:49,349] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-10-28 21:44:49,349] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-10-28 21:44:49,357] {taskinstance.py:1397} INFO - Executing <Task(_PythonDecoratedOperator): extract> on 2022-09-01 00:00:00+00:00
[2022-10-28 21:44:49,361] {standard_task_runner.py:52} INFO - Started process 379 to run task
[2022-10-28 21:44:49,363] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'get_files_cetram', 'extract', 'scheduled__2022-09-01T00:00:00+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/get_raw_files/get_files_denue.py', '--cfg-path', '/tmp/tmpcrki6ys2', '--error-file', '/tmp/tmp522ym9q6']
[2022-10-28 21:44:49,364] {standard_task_runner.py:80} INFO - Job 3: Subtask extract
[2022-10-28 21:44:49,394] {task_command.py:371} INFO - Running <TaskInstance: get_files_cetram.extract scheduled__2022-09-01T00:00:00+00:00 [running]> on host c6a5358eaead
[2022-10-28 21:44:49,533] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Freakisimo
AIRFLOW_CTX_DAG_ID=get_files_cetram
AIRFLOW_CTX_TASK_ID=extract
AIRFLOW_CTX_EXECUTION_DATE=2022-09-01T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-09-01T00:00:00+00:00
[2022-10-28 21:44:49,897] {python.py:173} INFO - Done. Returned value was: []
[2022-10-28 21:44:49,913] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=get_files_cetram, task_id=extract, execution_date=20220901T000000, start_date=20221028T214449, end_date=20221028T214449
[2022-10-28 21:44:49,935] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-10-28 21:44:49,996] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
